scripts/fastlane/Fastfile

```diff
@@ -0,0 +1,128 @@
+RELEASE_TAG_PREFIX = 'release_v'
+default_platform(:android)
+
+platform :android do |options|
+  gradle_properties_path = File.expand_path("#{Dir.pwd()}/../../gradle.properties")
+  gradle_project_root = File.dirname(gradle_properties_path)
+  change_log_path = "#{gradle_project_root}/CHANGELOG.md"
+  version_regex = /(\d*)\.(\d*)\.(\d*)(?:-unstable\.)?(\d*)?/
+  pr_body = %Q(
+## Release PR review checklist
+- [ ] Verify version name in gradle.properties
+- [ ] Verify CHANGELOG.md
+)
+
+  desc "Fetch all tags to ensure accurate calculation of the next release number."
+  before_all do 
+    sh('git', 'fetch', '--tags')
+    sh('git', 'fetch')
+  end
+
+  lane :configure_git_options do |options|
+    sh('git', 'config', 'user.email', options[:git_user_email])
+    sh('git', 'config', 'user.name', options[:git_user_name])
+  end
+
+  lane :create_next_release_pr do |options|
+    product_name = options[:product_name]
+    last_release_tag = last_git_tag(pattern: "#{RELEASE_TAG_PREFIX}*")
+    next_version, commits = calculate_next_release_version(release_tag_prefix: RELEASE_TAG_PREFIX, from_tag: last_release_tag)
+    UI.message("Version change #{last_release_tag} => #{next_version}")
+    update_gradle_properties(version: next_version)
+    releases = { 
+      feat: "minor",
+      fix: "patch", 
+      chore: "patch",
+      refactor: "patch",
+      perf: "patch",
+      test: "patch",
+      docs: "patch",
+      no_type: "patch"
+    }
+
+    # The analyze_commits function is a pre-requisite to running conventional_changelog. However, the versioning logic
+    # used by analyze_commits increments the version number once for each commit that's part of the release. As a result,
+    # you end up "wasting" version numbers. For example, your last release is 1.1.2 and you have 5 bug fixes to go in the
+    # next release, the version calculated by analyze_commit would be 1.1.7. We'll just use the version number we
+    # get from calculate_next_release_version. We'll use semantic_release for generating the change log since it has
+    # a richer feature set.
+    isReleasable = analyze_commits(match: "#{last_release_tag}", codepush_friendly:[], releases: releases)
+    update_change_log(next_version: next_version, last_release_tag: last_release_tag, commit_format: 'default', debug: true)
+
+    git_add(path: [gradle_properties_path, change_log_path])
+    git_commit(path: [gradle_properties_path, change_log_path], message: "chore(release): #{product_name} - #{next_version}")
```

Comment:
This seems special enough that we could give it its own conventional commit prefix.  Then, I think it would be a little more straightforward to distinguish it as a trigger - what do you think?

```suggestion
    git_commit(path: [gradle_properties_path, change_log_path], message: "release: #{product_name} - #{next_version}")
```